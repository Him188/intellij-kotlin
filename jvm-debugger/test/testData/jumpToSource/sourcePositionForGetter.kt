package sourcePositionForGetter

interface Interface {
    val iInt get() = 10
}

abstract class AbstractClass {
    val aInt get() = 20
}

class TestClass : AbstractClass(), Interface {
    val myInt get() = 30
}

fun main() {
    val instance = TestClass()
    // JUMP TO SOURCE: instance.myInt
    // JUMP TO SOURCE: instance.aInt
    // JUMP TO SOURCE: instance.iInt
    //Breakpoint!
    println("")
}
