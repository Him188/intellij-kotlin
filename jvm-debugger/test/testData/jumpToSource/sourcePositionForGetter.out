LineBreakpoint created at sourcePositionForGetter.kt:21
Run Java
Connected to the target VM
sourcePositionForGetter.kt:21
Source position of instance.myInt is 12
Source position of instance.iInt is 4
Source position of instance.aInt is 8
Disconnected from the target VM

Process finished with exit code 0

